src/connectors/aws_account.py

```diff
@@ -0,0 +1,100 @@
+"""AWS Account List
+Lists your AWS Accounts
+"""
+
+from runners.helpers import db
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+import boto3
+import datetime
+
+CONNECTION_OPTIONS = [
+    {
+        'name': 'source_role_arn',
+        'title': "source role arn",
+        'prompt': "assumer role arn",
+        'type': 'str',
+        'required': True
+    },
+    {
+        'name': 'destination_role_arn',
+        'title': "destination role arn",
+        'prompt': "destination role arn",
+        'type': 'str',
+        'required': True
+    },
+    {
+        'name': 'destination_role_external_id',
+        'title': 'destination_role_external_id',
```

Comment:
these should have `"` since they'll be displayed to humans and let's make them human readable

Reply:
fixed